docs/upgrade-csi-driver.md

```diff
@@ -0,0 +1,182 @@
+# How to upgrade juicefs CSI Driver from v0.9.0 to v0.10.0
+
+Juicefs CSI Driver separated JuiceFS client from CSI Driver since v0.10.0. But the upgrade from v0.9.0 to 0.10.0 will cause all PVs become unavailable, we can upgrade one by one node to make the upgrade smooth.
```

Comment:
Added a section to upgrade whole cluster:

1. stop all applications
2. upgrade CSI driver
3. start all applications